docker-compose.yml

```diff
@@ -38,18 +38,28 @@ services:
     ports:
       - "6379"
 
-  elastic:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.10.0
+  opensearch-node1:
+    image: opensearchproject/opensearch:1.3.9
+    container_name: opensearch-node1
     environment:
-      - network.host=0.0.0.0
-      - http.cors.enabled=true
-      - http.cors.allow-origin=*
-      - http.max_content_length=10mb
-      - rest.action.multi.allow_explicit_index=false
-      - "ES_JAVA_OPTS=-Xms1024m -Xmx1024m"
-      - discovery.type=single-node
+      - cluster.name=opensearch-cluster
+      - node.name=opensearch-node1
+      - bootstrap.memory_lock=true # along with the memlock settings below, disables swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms1024m -Xmx1024m" # Set min and max JVM heap sizes to at least 50% of system RAM
+      - "DISABLE_INSTALL_DEMO_CONFIG=true" # disables execution of install_demo_configuration.sh bundled with security plugin, which installs demo certificates and security configurations to OpenSearch
+      - "DISABLE_SECURITY_PLUGIN=true" # disables security plugin entirely in OpenSearch by setting plugins.security.disabled: true in opensearch.yml
```

Comment:
I meant to respond to this earlier, I was working on getting this, and will toss something in. Likely in a separate PR to avoid holding this up since prod and local are running difference services as of now.